script.js

```diff
@@ -1,4 +1,29 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  if ((input == 'scissors') || (input == 'stone') || (input == 'paper')) {
+  var myOutputValue = getResult(input);
   return myOutputValue;
+  };
+  return "The input options are scissors, stone or paper. Kindly select from these";
 };
+
+var getResult = function(input){
+  console.log ('you have input ' + input);
+  var arrayItems = ['stone','paper','scissors'];
+  var random = Math.floor(Math.random()*3);
+  var value = arrayItems[random];
```

Comment:
Might be good to name the variables to more accurately reflect its content, in this case, e.g. var handArray, randomInteger, chosenHand etc.